apps/frontend/src/pages/ProofPage/effects.tsx

```diff
@@ -41,9 +40,6 @@ export const useProductPageEffects = () => {
     generatorId: data?.certificate?.generatorId ?? '',
     energySource: <FuelType fuelType={data?.certificate?.energySource as FuelTypeEnum} />,
     region: `${data?.certificate?.country}, ${data?.certificate?.region}`,
-    redemptionDate: dayjs(data?.certificate?.redemptionDate).isValid()
```

Comment:
I think this should be replaced by `data?.redemptionDate` and not removed

Reply:
Totally yeah, forgot to change that. Thanks.

Reply:
Done.